### ISSUE TITLE
Another issue with null coalesce inspection and references

### ISSUE THREAD

**Issue by iakoder** [kalessil/phpinspectionsea#1127]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Bug                          |
| Plugin         | Php Inspections (EA Extended), 3.0.8.1 |
| Language level | PHP 7.0                        |

Not sure if this is the same as the other issue with references.  The following was erroneously flagged as a potential candidate for the null coalesce operator.

```
        if (isset($this->_xmlreturn[$this->_listtype]['fieldmap'])) {
            $objFields =& $this->_xmlreturn[$this->_listtype]['fieldmap'];
        } else {
            $objFields =& $this->_xmlreturn[$this->_listtype];
        }
```

and fixed to 

```
        $objFields = $this->_xmlreturn[$this->_listtype]['fieldmap'] ?? $this->_xmlreturn[$this->_listtype];
```
